app/webpacker/javascript/controllers/community_activity_component_controller.js

```diff
@@ -32,13 +33,41 @@ export default class extends ApplicationController {
   connect() {
     this.trackUnsavedChanges()
     this.checkForEvidence();
+    this.trackCharacterCount()
+  }
+
+  trackCharacterCount() {
+    this.updateCharacterCount = (element, index) => {
+      const remainingCharacters = this.minimumEvidenceLengthValue - element.value.length
+
+      if(element.value.length > 0 && element.value.length < this.minimumEvidenceLengthValue) {
+        this.characterCountMessageTargets[index].textContent = `Provide at least ${remainingCharacters} more characters`
+      } else {
+        this.characterCountMessageTargets[index].textContent = ""
+      }
+
+    };
+
+    this.textareaTargets.forEach((input, index) => {
+      input.addEventListener("input", () => this.updateCharacterCount(input, index))
```

Comment:
We should add a disconnect for this, to ensure the listener is removed when we remove the controller to keep the browser session tidy.